scipy/stats/_continuous_distns.py

```diff
@@ -8200,15 +8200,14 @@ class truncnorm_gen(rv_continuous):
     Notes
     -----
     The standard form of this distribution is a standard normal truncated to
-    the range [a, b] --- notice that a and b are defined over the domain of the
-    standard normal.  To convert clip values for a specific mean and standard
-    deviation, use::
-
-        a, b = (myclip_a - my_mean) / my_std, (myclip_b - my_mean) / my_std
-
-    `truncnorm` takes :math:`a` and :math:`b` as shape parameters.
-
-    %(after_notes)s
+    the range ``[a, b]``, where ``a`` and ``b`` are user-provided shape
+    parameters. The parameter ``loc`` shifts the mean of the underlying normal
+    distribution, and ``scale`` controls the standard deviation of the
+    underlying normal, but ``a`` and ``b`` are still defined over the domain
```

Comment:
```suggestion
    underlying normal, but ``a`` and ``b`` are still defined on the domain
```
It said "over" before. Would "on" be preferable? Or "with respect to"?

Reply:
I think I like "with respect to" better, and maybe we need a different word than "domain". "over" and "on" some "domain" make me think of the domain as a set, and technically, the sets _as sets_ are identical (i.e. whole real number line). "still defined with respect to the *standard* normal" might work better.